modules/standard/Regex.chpl

```diff
@@ -564,7 +568,7 @@ record regex : serializable {
                             for more details about error codes.
    */
   proc init(pattern: ?t, posix=false, literal=false, noCapture=false,
-            /*i*/ ignoreCase=false, /*m*/ multiLine=false, /*s*/ dotAll=false,
```

Comment:
Could we append to the documentation for nonGreedy that the setting is ignored when `posix=true` since it isn't compatible?

Reply:
Yes, of course, I just forgot to do so. Thanks for noticing.